### DISCUSSION TITLE
Cant connect to localhost

### DISCUSSION THREAD

**Discussion by MrMystery10-del** [tauri-apps/tauri#6829] [Q&A]
When i try to run my new generated project i get that error:
![Screenshot 2023-05-01 153254](https://user-images.githubusercontent.com/69641472/235458776-f5b0a269-60aa-4e64-b708-e213fd853e6d.png)
Im using `React/JavaScript/npm`

**Comment by Lakshya0257** [comment]
Yes, same thing happens with me....and now I am stuck

**Reply by FabianLars** [reply]
Typically this error means that the `distDir` is incorrectly configured (or more generally, that the frontend wasn't included in the app)

**Reply by Lakshya0257** [reply]
Yes, but in my case, my application was depending on a server that will run continuous. Because of that I was getting error :)
